apps/jobs/views.py

```diff
@@ -304,6 +304,34 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
             request.data["is_public"] = (
                 True if challenge_phase.is_submission_public else False
             )
+        else:
+            request.data["is_public"] = json.loads(request.data["is_public"])
+            if request.data.get("is_public") and challenge_phase.is_restricted_to_select_one_submission:
+                # handle corner case for restrict one public lb submission
+                try:
```

Comment:
Why do we need `try/except` here with a blank except? I might be missing something here.

Reply:
It was needed when using `.get` now it isn't